test/functional/cmdLineTests/CDSAdaptorTest/playlist.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+Copyright (c) 2020, 2020 IBM Corp. and others
+
+This program and the accompanying materials are made available under
+the terms of the Eclipse Public License 2.0 which accompanies this
+distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+or the Apache License, Version 2.0 which accompanies this distribution and
+is available at https://www.apache.org/licenses/LICENSE-2.0.
+
+This Source Code may also be made available under the following
+Secondary Licenses when the conditions for such availability set
+forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+General Public License, version 2 with the GNU Classpath
+Exception [1] and GNU General Public License, version 2 with the
+OpenJDK Assembly Exception [2].
+
+[1] https://www.gnu.org/software/classpath/license.html
+[2] http://openjdk.java.net/legal/assembly-exception.html
+
+SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+<playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../../TKG/playlist.xsd">
+	<test>
+		<testCaseName>CDSAdaptorTest</testCaseName>
+		<variations>
+			<variation>NoOptions</variation>
+		</variations>
+		<command>cp $(Q)$(TEST_RESROOT)$(D)cdsadaptortest.jar$(Q) .; \
+		cp $(Q)$(LIB_DIR)$(D)org.eclipse.osgi_3.16.100.v20200904-1304.jar$(Q) .; \
+		true CacheManagement.jar ; \
+		$(Q)$(TEST_JDK_HOME)$(D)bin$(D)jar$(EXECUTABLE_SUFFIX)$(Q) xf cdsadaptortest.jar; \
+		$(CONVERT_TO_EBCDIC_CMD) \
+		$(JAVA_COMMAND) $(JVM_OPTIONS) \
+		-DJARPATH=$(Q)$(TEST_RESROOT)$(D)cdsadaptortest.jar$(Q) \
+		-DEXE=$(SQ)$(JAVA_COMMAND) $(JVM_OPTIONS) $(SQ) \
+		-DCPDL=$(Q)$(P)$(Q) -DPATHSEP=$(Q)$(D)$(Q) \
+		-jar $(CMDLINETESTER_JAR) -config $(Q)$(TEST_RESROOT)$(D)cdsadaptortest.xml$(Q) \
+		-verbose -nonZeroExitWhenError; \
+		$(TEST_STATUS)</command>
+		<subsets>
+			<subset>8</subset>
```

Comment:
Does this test only work for java8?

Reply:
Tested on jdk11 j9 and ibm, jdk13 j9, and jdk14 j9, all passed, so enabled it for all jdk versions.